src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java

```diff
@@ -5,7 +5,7 @@
  */
 package com.microsoft.spring.data.gremlin.common;
 
-public class Constants {
+public abstract class Constants {
```

Comment:
I don't see any benefits of making it abstract. What's the motivation behind this?

Reply:
Some static analyzer may complain about the class has not default constructor, So we may need to add one for that. For avoiding this, and this class may not need any instance. Abstract may be a better option here.

Reply:
This is the convention for this situation of most java developers ? If so, just keep original code is OK. 
**PS**: Sorry I am not familiar with this. I think private constructor may looks kind of tricky and misleading. That's why I changed to abstract.